spec/dmv_spec.rb

```diff
@@ -32,12 +36,66 @@
       @facility_2.add_service('Written Test')
       @facility_3.add_service('New Drivers License')
       @facility_3.add_service('Road Test')
-
+      
       @dmv.add_facility(@facility_1)
       @dmv.add_facility(@facility_2)
       @dmv.add_facility(@facility_3)
-
+      
       expect(@dmv.facilities_offering_service('Road Test')).to eq([@facility_2, @facility_3])
     end
   end
+  
+  describe '#create_facility_helper' do
+    it 'converts co facility information from API correct format' do
+      co_facility_info = @dmv.create_facility_helper(@co_dmv_office_locations)
+      expect(co_facility_info.class).to eq(Array)
+      expect(co_facility_info.first).to eq({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+    end
+    
+    it 'converts ny facility information from API correct format' do
+      ny_facility_info = @dmv.create_facility_helper(@ny_dmv_office_locations)
+      expect(ny_facility_info.class).to eq(Array)
+      expect(ny_facility_info.first).to eq({name: 'EVANS COUNTY OFFICE', address: '6853 ERIE RD DERBY NY 14006', phone: '(716) 858-7450'})
+    end
+
+    it 'converts MO facility information from API correct format' do
+      mo_facility_info = @dmv.create_facility_helper(@mo_dmv_office_locations)
+      expect(mo_facility_info.class).to eq(Array)
+      expect(mo_facility_info.first).to eq({name: 'OAKVILLE', address: '3164 TELEGRAPH ROAD ST LOUIS MO 63125', phone: '(314) 887-1050'})
+    end
+  end
```

Comment:
These are great tests. One suggestion I have is to verify (in each example) that _every_ element of the returned array has the expected data types:
```
mo_facility_info = @dmv.create_facility_helper(@mo_dmv_office_locations)
mo_facility_info.each do |facility|
  expect(facility).to be_a(Hash)
  expect(facility[:name]).to be_a(String)
  # etc
end
```